utils/actions/labelPullRequest.ts

```diff
@@ -0,0 +1,61 @@
+const { Configuration, OpenAIApi } = require("openai");
+import { Octokit } from "octokit";
+
+const configuration = new Configuration({
+  apiKey: process.env.OPEN_AI_KEY,
+});
+const openai = new OpenAIApi(configuration);
+export default function flagPullRequest({
+  prTitle, 
+  businessLogicSummary,
+  github_token,
+  owner,
+  repo,
+  issue_number
+}: {
+  prTitle?: string;
+  businessLogicSummary?: string;
+  github_token?: string;
+  owner: string;
+  repo: string;
+  issue_number: number
+}) {
+
+  if (!github_token) {
+    return { error: "no github token" };
+  }
+
+  const octokit = new Octokit({
+    auth: github_token
+  });
+
+
+  const prompt = `The goal of this PR is to: ${prTitle}. \n The information related to this PR is: ${businessLogicSummary}. \n On a scale of 1(very different)-10(very similar), how similar the PR's goal and the PR's related information are? Take into account semantics. Don't explain your reasoning, just print the rating. Don't give a range for the rating, print a single value.`
```

Comment:
Any tracking?

Reply:
Additional? No. 

That's already happening in line 551 of this file. We're not adding an additional event. We're expanding what an event does.

Reply:
Which file?
I see this file has about sixty lines

Reply:
Sorry I meant utils/actions/labelPullRequest.ts which calls labelPullRequest and subsequently the PostHog tracking.